### DISCUSSION TITLE
Official Homebrew GitHub Actions in the GitHub Marketplace

### DISCUSSION THREAD

**Discussion by rgoldberg** [Homebrew/discussions#5716] [Everyday usage] [answered]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

Is there a reason why the [official Homebrew GitHub Actions](https://github.com/Homebrew/actions) are not available from the [GitHub Marketplace](https://github.com/marketplace?query=homebrew)?

**Comment by SMillerDev** [comment, marked as answer]
Because most, if not all, of them are for use by Homebrew and are not supported outside the Homebrew CI setup.
